lib/services/productService.ts

```diff
@@ -0,0 +1,30 @@
+import { Product } from '../models/product';
+import { ProductRepository } from '../repositories/productRepository';
+
+export class ProductService {
+    constructor(private repo: ProductRepository) {}
```

Comment:
Should've just initialized the repository here.

Reply:
done